grafana/deployment.yaml

```diff
@@ -47,6 +65,12 @@ spec:
             name: grafana-storage
           - mountPath: /etc/grafana
             name: grafana-configmap-volume
+          - mountPath: /etc/grafana/provisioning/datasources
+            name: grafana-datasources-configmap-volume
+          - mountPath: /grafana-tokens
```

Comment:
This shouldn't have to be mounted in the grafana container if you're moving the relevant information to `grafana-shared`, right?